src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,46 @@
+.nav {
+  padding: $padding-nav;
+  margin-right: 50px;
+
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    text-decoration: none;
+    color: $color-main-accent;
```

Comment:
remove text-decoration, you wrote it in reset file

Reply:
**Fixed**